testsuite/pytests/mpi/4/test_set_status_resolution_nthreads.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# test_set_status_resolution_nthreads.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+import nest
+import pytest
+
+
+@pytest.fixture
+def reset():
+    nest.ResetKernel()
+
+
+def testSetStatus_resolution_before_nthreads(reset):
+    """"""
```

Comment:
Should these tests have docstrings?

Reply:
They should and doctrings have been added in 90e6b6585.